samples/BasicTest.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
`BasicTest.ipynb` is not an appropriate name. Let's use meaningful names and folders for the sample files. Something like samples/inmemory/memory_db.ipynb.